libgnucash/engine/test-core/test-engine-stuff.cpp

```diff
@@ -2032,7 +2022,7 @@ make_trans_query (Transaction *trans, TestQueryTypes query_types)
         xaccQueryAddAccountGUIDMatch (q, list, QOF_GUID_MATCH_ANY, QOF_QUERY_AND);
 
         for (node = list; node; node = node->next)
-            g_free (node->data);
+            guid_free (static_cast<GncGUID*>(node->data));
         g_list_free (list);
```

Comment:
oops sorry this one too!

Reply:
My fault -- I should have checked more carefully. Thanks.